requirements.txt

```diff
@@ -1,4 +1,4 @@
 Pillow
-ModernGL
+moderngl==5.5.4
```

Comment:
here we have to do something more generic, dynamic ... maybe a grid of version, and multiple build targets ... something like this ... dunno :p

Reply:
`moderngl<6` should work for now

Reply:
![image](https://user-images.githubusercontent.com/7001058/73614601-e5f1e100-4600-11ea-9797-4ac2b69426c3.png)
Seems not to work :/

Reply:
Did this work with 5.5.4 or something? The only difference is that we're loading libGL dynamically

Reply:
Yep you can see on the CI that's working with `moderngl<5.6.0`
here for example: https://github.com/yoyonel/Headless-rendering-with-python/runs/422066909?check_suite_focus=true

Reply:
Oki i found the problem, in the container (github-action container) there is no link (symbolic) to `libGL.so` ... MmMmm .. i will add a symbolic link into the pipeline, i don't know if it's a good practice ... i have to search more.

Reply:
`ctx.create_context(..)` also supports passing in the `libgl='libGL.so.1'` if overrides are needed. Whatever works ;)

Reply:
![image](https://user-images.githubusercontent.com/7001058/73656721-07021280-4691-11ea-8c95-d2ec94332d08.png)
yeahhhhhhh ^^